client/src/pages/ChefSearch.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState, useContext } from "react";
+import ResponsiveSideBar from "../components/ResponsiveSideBar";
+import ChefCard from "../components/ChefCard";
+import Main from "../components/Main";
+import { Typography, Button, Grid, TextField } from "@material-ui/core";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import { UserContext } from "../contexts/user/UserContextProvider";
+import { abortableFetch, consoleErrorNonAbortErrors } from "../utils";
+import useCuisineSelector from "../lib/useCuisineSelector";
+
+const useStyles = makeStyles({
+    availableChefs: { paddingBottom: "15px", display: "inline", fontSize: "30px" },
+});
+
+function ChefSearch(props) {
+    const classes = useStyles();
+    const {
+        ALL,
+        cuisines,
+        availableCuisines,
+        addCuisine,
+        removeCuisine,
+    } = useCuisineSelector();
+    const { profile } = useContext(UserContext);
+    const {
+        primaryAddress: { city, region, country },
+    } = profile;
+    const location = [city, region, country].join(", ");
+    const [radiusKm, setRadiusKm] = useState(100);
+    const [results, setResults] = useState({
+        chefs: [],
+        chefMeals: {},
```

Comment:
Are we using this anywhere?

Reply:
refactored it to just use chefs array and removed the rest of the object